From 6df136cc1f7ec5d8286bc58becf6672fa716f1fb Mon Sep 17 00:00:00 2001
From: Samuel Thibault <sthibault@debian.org>
Date: Wed, 4 Aug 2010 13:34:05 +0300
Subject: Make hints/gnu.sh append to $ccflags rather than overriding them

Bug-Debian: http://bugs.debian.org/587901

Don't override possible extra $ccflags values given to Configure
on GNU/Hurd.

Patch-Name: fixes/hurd-ccflags.diff
---
 hints/gnu.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hints/gnu.sh b/hints/gnu.sh
index 2cfce54..c1ba2db 100644
--- a/hints/gnu.sh
+++ b/hints/gnu.sh
@@ -19,7 +19,7 @@ lddlflags='-shared'
 ccdlflags='-Wl,-E'
 
 # Debian bug #258618
-ccflags='-D_GNU_SOURCE'
+ccflags="-D_GNU_SOURCE $ccflags"
 
 # The following routines are only available as stubs in GNU libc.
 # XXX remove this once metaconf detects the GNU libc stubs.
